_data/fellows.yml

```diff
@@ -1,89 +1,82 @@
 - name: Sahiba Khan
   img: assets/img/sahiba.png
-  birth_date: 11 July 
+  username: sahibardj
+  birth_date: 11 July
   location: Agra, India
   school: G L Bajaj
   tech_skills: Go, C++, SQL
   hobbies: writing
   bio: Heyya, I'm Sahiba a software engineer driven by solving real world problems and making a tangible change.
   url: https://github.com/sahibardj
-  social:
-      twitter: https://twitter.com/
-      github: https://github.com/sahibardj
-      linkedin: https://www.linkedin.com/in/sahiba-khan-rdj/
+  twitter: https://twitter.com/
+  github: https://github.com/sahibardj
+  linkedin: https://www.linkedin.com/in/sahiba-khan-rdj/
   coordinates:
-      lat: 27
-      lng: 78
-
-- name: Amanda
-  img: amanda.jpg
-  location: New York City, USA
-
-- name: Aaron
-  img: aaron.jpg
-  location: London, UK
-
-- name: Swift
-  img: swift.jpg
-  location: New York City, USA
-
-- name: Arpit Bhalla
-  img: mlh.jpg
-  username: arpitBhalla
-  location: New York City, USA
-  url: https://github.com/arpitbhalla
-  social:
-    twitter: arpit
-    github: arpit
-    linkedin: arpit
-  
+    lat: 27
+    lng: 78
 
 - name: Akhilesh
   username: AkhileshManda
-  img: AkhileshManda.jpg
+  img: /assets/img/AkhileshManda.jpg
   location: Bengaluru, IN
   url: https://github.com/AkhileshManda
-  social:
-      twitter: https://twitter.com/Akhiles63114004
-      github: https://github.com/AkhileshManda
-      linkedin: https://www.linkedin.com/in/akhilesh-manda-7a9b661b8/
-  
+  twitter: https://twitter.com/Akhiles63114004
+  github: https://github.com/AkhileshManda
+  linkedin: https://www.linkedin.com/in/akhilesh-manda-7a9b661b8/
+
 - name: Kaushal
   username: kaushal
-  img: KaushalJoshi.png
+  img: /assets/img/KaushalJoshi.png
```

Comment:
Hey! Why have you added /assets/img/? I think it'll be uniform just to have the name of the file and while using the images somewhere, we can add the /assets/img/ path